### ISSUE TITLE
jibri does not work at stable-5963

### ISSUE THREAD

**Issue by JacBra** [jitsi/docker-jitsi-meet#1075]
I created a brand new Jitsi docker stack following https://jitsi.github.io/handbook/docs/devops-guide/devops-guide-docker.
However, Jibri recordings do not function out of the box.
I have set the PUBLIC_URL variable to an external https URL.

**Comment by saghul** [comment]
Please provide some logs, otherwise there js no way to help you.

**Comment by JacBra** [comment]
Hi, I attached several logs. Please let me know if you would like to have more. Thanks for your time to help me out.

[log.0.txt](https://github.com/jitsi/docker-jitsi-meet/files/6649409/log.0.txt)
[chrome-browser-log.log](https://github.com/jitsi/docker-jitsi-meet/files/6649416/chrome-browser-log.log)
[all-docker-containers-logs.log](https://github.com/jitsi/docker-jitsi-meet/files/6649439/all-docker-containers-logs.log)

[docker-compose.yml.txt](https://github.com/jitsi/docker-jitsi-meet/files/6649451/docker-compose.yml.txt)
[jibri.yml.txt](https://github.com/jitsi/docker-jitsi-meet/files/6649456/jibri.yml.txt)
[my .env.txt](https://github.com/jitsi/docker-jitsi-meet/files/6649464/my.env.txt)

**Comment by tusharsonawanes** [comment]
Did you find any solution to this ?

**Comment by mhow2** [comment]
@JacBra : I had a look at your `.env`, it seems that the secrets haven't been generated as advised in the documentation you have linked (ie. `./gen-passwords.sh`).

**Comment by Bomjman** [comment]
Same issue

**Comment by JacBra** [comment]
> 
> 
> @JacBra : I had a look at your `.env`, it seems that the secrets haven't been generated as advised in the documentation you have linked (ie. `./gen-passwords.sh`).

@mhow2 : I did that on purpose not to expose my passwords here. On my system, the passwords have been generated as advised in the documentation.

**Comment by mhow2** [comment]
> > @JacBra : I had a look at your `.env`, it seems that the secrets haven't been generated as advised in the documentation you have linked (ie. `./gen-passwords.sh`).
> 
> @mhow2 : I did that on purpose not to expose my passwords here. On my system, the passwords have been generated as advised in the documentation.

Fair enough - Sorry, I thought about this possibility only afterwards.

On my side, for some reason Jibri started working only after I have turned on Let's Encrypt feature against a real domain name (and adjusted `PUBLIC_URL`), which you have already done it seems.

Also I don't the jibri error that you have on your side:

```
java.lang.RuntimeException: org.jivesoftware.smack.XMPPException$XMPPErrorException: XMPP error reply received from jibribrewery@internal-muc.meet.jitsi/jibri-instanse-826424585: XMPPError: not-allowed - cancel
	at org.jitsi.xmpp.mucclient.MucClient$1.authenticated(MucClient.java:287)
```

Instead I have a normal:
```2021-06-29 12:40:15.234 INFO: [27] org.jitsi.xmpp.mucclient.MucClient.log() Joined MUC: jibribrewery@internal-muc.meet.jitsi```

Could it be related to prosody ? A wrong password ?

**Comment by tusharsonawanes** [comment]
Hi, try adding

```
```yaml
networks:
            meet.jitsi:
        extra_hosts:
           - "${HOST}:${DOCKER_HOST_ADDRESS}"
```

Where HOST-> public url except https://
and DOCKER_HOST_ADDRESS -> IP address of the host

**Comment by HthSolid** [comment]
that solved it for me

**Comment by JacBra** [comment]
@tusharsonawanes : thanks so much, that works so I have a workaround now!
So the reason probably is that the Jibri container is trying to connect to the PUBLIC_URL and it can't resolve it, right? 

@saghul: Is this something that should be fixed somehow?

**Comment by saghul** [comment]
The public URL is meant to be public and resolvable. If it isn't, your workaround works, but that's not something that will be part of the base image.
